source/outlines/2016-03-08.markdown

```diff
@@ -0,0 +1,70 @@
+---
+title: 20160308
+layout: outline
+---
+
+## All
+
+### Word of the Day
+
+* qua - preposition - in the capacity or character of; as
+
+
+## 1508
+
+### Final Assessment (9:00 - YOU FINISH)
+
+It's the [Final Assessment](https://www.youtube.com/watch?v=9jK-NcRmVcw)
+
+
```

Comment:
Since most of the class won't be doing the final assessment - @jmejia do you want to do a repeat of 'pairing with Mod 3' - most of the 1508 crew is pretty positive about it!